src/main/java/eventprocessing/models/SqsResponse.java

```diff
@@ -0,0 +1,66 @@
+package eventprocessing.models;
+
+import com.google.gson.annotations.SerializedName;
+import org.joda.time.DateTime;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class SqsResponse {
+    @SerializedName("MessageId")
+    public String messageId;
+    public Message message;
+    @SerializedName("Message")
+    public String messageString;
+    @SerializedName("Timestamp")
+    public String responseTimestamp;
+
+    @Override
+    public String toString() {
+        if (message != null) {
+            return "messageId: " + messageId + "\nMessage: " + String.valueOf(message.getLocationId());
+        } else {
+            return "messageId: " + messageId + "\nMessage: NULL";
+        }
+    }
+
+    public void setMessage(Message message) {
```

Comment:
I just saw your comment on https://github.com/MatthewBurstein/EventProcessing/pull/1#discussion_r198131503 - I still think that having a `setMessage` method is hard to follow. If you need to deserialize the AWS response and the message separately then that's OK, but trying to combine them with a `setter` is confusing.